docs/documentation/api.md

```diff
@@ -334,14 +334,20 @@ above.
    post an example of the output in the pull request.
 
 1. Run `./tools/create-api-changelog` which will create a new empty
-   changelog file in `api_docs/unmerged.d/` directory. Open this
+   markdown file in `api_docs/unmerged.d/` directory. The file will be
+   named in the format `ZF-...` (e.g., `ZF-1f4a39.md`). Open this
```

Comment:
Is noting the prefix here super useful? It might be more helpful to note that the path to the new file is printed to the terminal with an example of the path there, e.g. `api_docs/unmerged.d/ZF-1f4a39.md`).